ts/sagas/onboarding.ts

```diff
@@ -100,6 +104,23 @@ function* tosCheckSaga(): Iterator<Effect> {
     // Here we wait the user accept the ToS
     yield take(TOS_ACCEPT_REQUEST);
 
+    // Get the current user profile from the store
+    const userProfile: ProfileState = yield select(profileSelector);
+
+    // We have the profile info but the user hasn't a profile active on API
```

Comment:
I don't understand this, what's the difference between profile info and active profile?

Reply:
The backend return a Backend Profile that is the union of SPID Profile (fiscal_code, family_name, ....) and API Profile (is_inbox_enabled, is_webhook_enabled)

Inside this Backend Profile there is a `has_profile` boolean that is true only if the user already has an API Profile (the backend query the API for this info).
So:

- `We have the profile info` means that we fetched the Backend Profile
- `but the user hasn't a profile active on API` means that Backend Profile `has_profile` is false (no profile on API) and we need to create it (with inbox and webhook enabled).

Reply:
ok I see, makes sense